db/migrations/20250121204110_insert_data_from_json.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class InsertDataFromJson extends AbstractMigration
+{
+    public function up()
+    {
+        $usersData = json_decode(file_get_contents(__DIR__ . '/../data/users.json'), true);
+        $studentsData = json_decode(file_get_contents(__DIR__ . '/../data/students.json'), true);
+        $meetingsData = json_decode(file_get_contents(__DIR__ . '/../data/meetings.json'), true);
+
+        $usersTable = $this->table('users');
+        $studentsTable = $this->table('students');
+        $meetingsTable = $this->table('meetings');
+
+        foreach ($usersData as $user) {
+            $usersTable->insert([
+                'id' => $user['id'],
+                'email' => $user['email'],
+                'password' => $user['password'],
+                'role' => $user['role'],
+            ])->save();
+        }
+
+        foreach ($studentsData as $student) {
+            $studentsTable->insert([
+                'id' => $student['id'],
+                'fname' => $student['fname'],
+            ])->save();
+        }
+
+        foreach ($meetingsData as $meeting) {
+            $meetingsTable->insert([
+                'id' => $meeting['id'],
+                'student_id' => $meeting['student_id'],
+                'topic' => $meeting['topic'],
+                'body' => $meeting['body'],
+                'user_id' => $meeting['user_id'],
+                'created_at' => $meeting['created_at'],
+                'updated_at' => $meeting['updated_at'],
+                'meeting_datetime' => $meeting['meeting_datetime'],
+            ])->save();
+        }
```

Comment:
Phinx умеет вставлять целый массив данных одним запросом, при этом выполняется всего один запрос в БД. Если ты используешь foreach и у тебя например будет 1к строк данных - оно выполнит 1к запросов в БД и это дает большую нагрузку на БД. 
Предлагаю сразу переделать чтоб привыкать к хорошим практикам. 
```suggestion
$usersTable->insert($usersData)->save();
$studentsTable->insert($studentsData)->save();
$meetingsTable->insert($meetingsData)->save();
```